crates/bevy_transform/src/components/global_transform.rs

```diff
@@ -74,11 +74,36 @@ impl GlobalTransform {
         Mat4::from_scale_rotation_translation(self.scale, self.rotation, self.translation)
     }
 
+    #[inline]
+    pub fn right(&self) -> Vec3 {
+        self.rotation * Vec3::unit_x()
```

Comment:
I'm confused... if Bevy is left-handed, and we're calling +z forward, then doesn't get make +x _left_?